elements/lisk-chain/test/unit/process.spec.ts

```diff
@@ -480,7 +489,10 @@ describe('blocks/header', () => {
 
 			it('should not call apply for the transaction and throw error', async () => {
 				// Arrange
-				const stateStore = new StateStore(storageStub);
+				const stateStore = new StateStore(storageStub, {
+					lastBlockHeaders: [],
+					networkIdentifier: 'network-identifier-chain-1',
+				});
```

Comment:
since i see everywhere we are creating new instance, this could be wrapped in a `beforeAll` hook?

Reply:
it is injecting specific storage stub everytime, so it shouldnt be in the hook i think